dts/bindings/usb/nxp,mcux-usbd.yaml

```diff
@@ -6,7 +6,7 @@ description: |
 
 compatible: "nxp,mcux-usbd"
 
-include: usb-ep.yaml
+include: [usb-ep.yaml, pinctrl-device.yaml]
```

Comment:
This `pinctrl-device.yaml` should be updated in the commit enabling pinctrl for the USB driver